app/api/threads/list_threads.feature

```diff
@@ -0,0 +1,172 @@
+      | Charlotte           | EtienneKleinCanWatchAnswer3           | Université           | waiting_for_participant |                      | EtienneKlein is_mine=0 -> List thread ok: can_watch >= answer                                                          |
+      | Charlotte           | EtienneKleinHasValidated5             | Université           | closed                  | 2021-12-20T00:00:00Z | EtienneKlein is_mine=0 -> List thread ok: part of helper group, closed thread for less than 2 weeks and validated item |
+      | Charlotte           | EtienneKleinCanWatchAnswer4           | Université           | closed                  | 2021-12-20T00:00:00Z | EtienneKlein is_mine=0 -> List thread ok: can_watch >= answer                                                          |
+      | Charlotte           | EtienneKleinHasValidated6             | Université           | closed                  | 2021-11-00T00:00:00Z | EtienneKlein is_mine=0 -> List thread notok: closed for more than 2 weeks                                              |
+      | Charlotte           | EtienneKleinCanWatchAnswer5           | Université           | closed                  | 2021-11-00T00:00:00Z | EtienneKlein is_mine=0 -> List thread ok: can_watch >= answer                                                          |
+      | Baptiste            | BaptisteCanViewInfo                   |                      |                         |                      | Baptiste is_mine=1 -> notok: can_view < content                                                                        |
+      | Baptiste            | BaptisteCanViewContent1               |                      |                         |                      | Baptiste is_mine=1 -> ok: can_view >= content                                                                          |
+      | Thibaut             | BaptisteCanViewContent2               |                      |                         |                      | Baptiste is_mine=1 -> notok: not the participant                                                                       |
+      | Baptiste            | BaptisteCanViewContentWithDescendants |                      |                         |                      | Baptiste is_mine=1 -> ok: can_view >= content                                                                          |
+      | DavidBowie          |                                       |                      |                         |                      |                                                                                                                        |
+    And the time now is "2022-01-01T00:00:00Z"
+
+  Scenario: Should have all the fields properly set, including first_name and last_name when the access is approved
+    Given I am MarieCurie
+    And there is a group Laboratory referenced by @Laboratory
```

Comment:
Doesn't gherkin rule usually quote string so that the reader know what is part of a static rule and what is a "variable"?

Reply:
It's up to the rule definition, it would make sense here to add " because it's not a natural sentence